src/material-experimental/mdc-list/list-base.ts

```diff
@@ -92,7 +93,14 @@ export abstract class MatListItemBase implements AfterContentInit, OnDestroy, Ri
               @Optional() @Inject(MAT_RIPPLE_GLOBAL_OPTIONS)
                   globalRippleOptions?: RippleGlobalOptions) {
     this._hostElement = this._elementRef.nativeElement;
-    this.rippleConfig = globalRippleOptions || {};
+    this.rippleConfig = {...(globalRippleOptions || {})};
```

Comment:
I'm a little iffy about this part. The problem with cloning it is that it won't be able to pick up dynamic changes to the `disabled` value anymore. I went with my current approach, because I believe that it's preferred to mutating a global object, but I'm open to alternative suggestions.

Reply:
I think we officially don't support the dynamic changes (though we've been landing changes in the past to make it work IIRC), so we should be good with that. Maybe add a small comment on why we clone here?

Alternatively we could use a getter for `rippleConfig` and support dynamic changes. Not feeling strongly.

Reply:
Leaving a comment sounds good 👍 I only noticed this since the dev app depends on it to disable ripples dynamically.